pending/contributing.pod

```diff
@@ -0,0 +1,122 @@
+community depends on B<you> to report bugs that you find along the way!
+
+=head2 IRC
+
+A great way to participate and contribute to the Dancer project is to be active
+on the IRC channel. The Dancer Core Team strives to maintain a friendly and civil
+environment on the channel at all times, and users of C<#dancer> would generally
+agree that it's a pretty friendly place to be.
+
+To get started, all you have to do is point your favorite IRC client to
+C<irg.perl.org#dancer> and you're off and running! Ask a question, answer
+a question, pitch your plugin idea... there are a number of ways for you 
+to get involved.
+
+All users are protected by our 
+L<community policy and community standards|https://metacpan.org/pod/distribution/Dancer2/lib/Dancer2/Policy.pod>
+document. The Core Team takes violations of this policy very, very seriously.
+
+=head2 Sending a Documentation Patch
+
+This is one of the easiest ways to get involved, and how yours truly got his
+start in the Dancer community (so I know for a fact this is an effective 
+way!). We as developers are not always the best at documenting our own work,
+no matter how hard we try. We are biased, and used to looking at our work 
+in a particular way, and we cannot always explain how something should be 
+used in the best way, or document things in such a way that it answers all
+the questons a user may have. Dancer2 is no exception.
+
+Did you find a particular area of the docs to be hard to understand? Clarify
+it and send us a pull request on GitHub. Did we forget to document a 
+method or configuration setting? Send us a patch! Dancer's documentation is 
+a living work, and will evolve and improve over time with the help of our
+community. 
+
+Another useful addition to the documentation is to add a recipe to the 
+L<cookbook|Dancer2::Cookbook>. It's a great place to show a useful tip that
+doesn't really fit elsewhere in the documentation.
+
+=head2 Taking an Issue from the GitHub Queue
+
+At the time of this writing, there are 82 current issues in the GitHub queue,
+13 of which are labelled Beginner Suitable. A great way to help out the 
+Dancer Core Team and the community is to pick up one of these issues, hack it
+out, and send us a pull request. No matter what your level of expertise is,
+there is something in the issue queue that will be suitable for you.
+
+Want to get your pull request looked at even quicker? By making sure your 
+pull request adequetely tests the issue you have tackled, it helps the Core
+Team fast track your work to the approval stage (because we don't have to
+write the tests...).
+
+We try to evaluate pull requests as quickly as we can. Please don't get 
+discouraged if it takes us a bit. A gentle reminder now and then is even ok,
+but remember we are all volunteers doing this for our love of Dancer too :)
+
+B<Note:> To get started hacking, make sure you have L<Dist::Zilla> installed.
```

Comment:
There's also `Dist::Zilla::PluginBundle::Dancer`, which has essential plugins we use.